src/backend/config.ts

```diff
@@ -59,8 +59,10 @@ const env = Environment();
 export const sessionConfig: ISessionKonfigurasjon = {
     cookieSecret: [`${process.env.COOKIE_KEY1}`, `${process.env.COOKIE_KEY2}`],
     navn: 'familie-ks-sak-v1',
-    redisPassord: process.env.REDIS_PASSWORD,
     redisUrl: env.redisUrl,
```

Comment:
Kan vi sende med dette 🤔
![image](https://github.com/navikt/familie-ks-sak-frontend/assets/17828446/c1494478-a9cc-4314-be15-931a6f04e30c)

Reply:
Det skal i utgangspunktet ikke ha noe å si dersom man ser på koden i `familie-backend`. Så lenge man sender inn `redisFullUrl` skal den bruke Aiven Redis uavhengig av om man også sender inn `redisUrl`. Den ligger også fortsatt inne i ef-sak-frontend etter at de gikk over til Aiven Redis da de heller ikke har ryddet bort den gamle redisen enda.

Kan forsåvidt fjerne den og validere på nytt i preprod, for den skal jo uansett bort på sikt 👍

Reply:
Det skal i utgangspunktet ikke ha noe å si dersom man ser på koden i `familie-backend`. Så lenge man sender inn `redisFullUrl` skal den bruke Aiven Redis uavhengig av om man også sender inn `redisUrl`. Den ligger også fortsatt inne i ef-sak-frontend etter at de gikk over til Aiven Redis da de heller ikke har ryddet bort den gamle redisen enda.

Kan forsåvidt fjerne den og validere på nytt i preprod, for den skal jo uansett bort på sikt 👍

Reply:
Fjernet `redisUrl` og fungerer fortsatt i preprod 👍